monitor.py

```diff
@@ -87,9 +89,12 @@ class Monitor(object):
     def __init__(self, _db, collection=None):
         self._db = _db
         cls = self.__class__
-        if not cls.SERVICE_NAME:
+        if self.SERVICE_NAME:
+            self.service_name = self.SERVICE_NAME
+        elif cls.SERVICE_NAME:
+            self.service_name = cls.SERVICE_NAME
```

Comment:
This allows a specific instance of a Monitor to define SERVICE_NAME rather than it needing to be defined ahead of time on the class level.

Reply:
I thought `self.SERVICE_NAME` would already be `cls.SERVICE_NAME` if it wasn't defined on the instance.

Reply:
I'm talking about the opposite situation -- cls.SERVICE_NAME is not defined but self.SERVICE_NAME is.

Reply:
Right, but I don't think you need the `elif` here - if `cls.SERVICE_NAME` is defined it becomes `self.SERVICE_NAME` automatically.